templates/modals/update_password.html

```diff
@@ -8,6 +8,11 @@
   on-submit="updatePassword()"
 >
   <form action="" method="POST">
+    <div class="form-group" ng-class="{'has-error': errors.currentPassword}">
```

Comment:
The new way to add the required `*` is to add a `required` class to this div, eg:

```html
<div class="form-group required" ng-class="{'has-error': errors.currentPassword}">
```

This adds the `*` through CSS with the right styling and spacing and so on.